mypy_self_check.ini

```diff
@@ -13,6 +13,10 @@ enable_error_code = ignore-without-code,redundant-expr
 enable_incomplete_feature = PreciseTupleTypes
 show_error_code_links = True
 
+[mypy-mypy.*]
+# TODO: enable for `mypyc` and other files as well
```

Comment:
I also did mypyc, it wasn't much extra work.

Reply:
Yeah, I prefer to split the PRs to be as small as possible. I will do that in the next one :)